service/src/main/java/com/ge/predix/acs/attribute/readers/ExternalAttributeReader.java

```diff
@@ -21,33 +21,45 @@
 import com.ge.predix.acs.rest.attribute.adapter.AttributesResponse;
 
 public abstract class ExternalAttributeReader implements AttributeReader {
+
+    @Value("${MAX_NUMBER_OF_ATTRIBUTES:1500}")
+    private int maxNumberOfAttributes;
+
+    @Value("${MAX_SIZE_OF_ATTRIBUTES:500000}")
```

Comment:
Use `MAX_SIZE_OF_ATTRIBUTES_IN_BYTES` (similar to the name of the member variable it annotates)